core/src/extras.rs

```diff
@@ -1,4 +1,4 @@
-use crate::{BlockNumber, Capacity};
+use crate::{EpochNumber, BlockNumber, Capacity};
```

Comment:
Items in `use` should be sorted.
(If you have `rust toolchain`, you can execute `cargo fmt -- --emit files` to fix this.)

Reply:
Holly I have thought about that but thought we would have a precommit hook. Updating.